src/form.ts

```diff
@@ -30,9 +66,49 @@ export class Form {
       textContent: 'Take a 2-minutes survey🙏',
     });
 
-    this.container.appendChild(titleContainer);
-    this.container.appendChild(descriptionContainer);
+    formCollapsed.appendChild(titleContainer);
+    formCollapsed.appendChild(descriptionContainer);
+    this.container.appendChild(formCollapsed);
 
     document.body.appendChild(this.container);
+
+    formCollapsed.addEventListener('click', () => {
+      if (this.container.className.includes(classes.open)) {
+        this.collapseWidget();
+      } else {
+        this.openWidget();
+      }
+    });
+  }
+
+  /**
+   * Create and add form to document
+   *
+   * @returns {HTMLFormElement}
```

Comment:
no need to add `@returns` to jsdoc as it's typescript and return type is already specified